inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java

```diff
@@ -283,12 +283,14 @@ private DataConfig getFileTaskFromEntity(TaskRequest taskRequest, StreamSourceEn
 
     private List<DataConfig> fetchIssuedTasks(TaskRequest taskRequest) {
         final String agentIp = taskRequest.getAgentIp();
+        final String agentClusterName = taskRequest.getAgentIp();
```

Comment:
Is the Agent IP also used as Agent cluster name?

Reply:
I fixed it.